vhdbuilder/packer/components.json

```diff
@@ -124,14 +127,17 @@
       ],
       "multiArchVersions": [
         "v1.0.0.2",
-        "v1.0.0.3"
+        "v1.0.0.3",
```

Comment:
is it expected to have 4 versions?

Reply:
1.0 and 1.1 are considered different minor versions, should be by design